open_spiel/python/mfg/algorithms/best_response_value.py

```diff
@@ -82,13 +82,16 @@ def eval_state(self, state):
       self._state_value[state_str] = (
           state.rewards()[0] + self.eval_state(new_state))
       return self._state_value[state_str]
-    else:
-      assert state.current_player() == 0, "The player id should be 0"
+    assert state.current_player() == pyspiel.PlayerId.DEFAULT_PLAYER_ID, (
+      "The player id should be 0")
+    state_legal_actions = state.legal_actions()
+    max_q = 0
+    if state_legal_actions:
```

Comment:
Will there be a case with no legal actions?

Reply:
Yes, for my implementation of routing game, one player might have no action (no routing choice) at a specific time step.